### DISCUSSION TITLE
Car charging from only excess battery storage

### DISCUSSION THREAD

**Discussion by andykorol1** [nathanmarlor/foxess_em#291] [Ideas]
So recently with the amount of sun I've wondered if we could charge the car in the scheduled window overnight but only use the excess in the battery and not draw any from the grid. In my instance this would only be 6kWh but this allows me to charge the house battery back up the day after with solar while the car is at work. Is this a fairly easy option to add in the future?

**Comment by nathanmarlor** [comment]
@andykorol1 this should already work - if you set the car to charge overnight it'll use whatever is in the battery until it hits Min SoC (On Grid), at which point it'll stop discharging the battery and start pulling from the grid.

**Comment by andykorol1** [comment]
It does exactly as you say but I was trying to find a way to stop the charge after the battery was at min SoC and not draw from the grid. Basically only using the excess solar from that day.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by dreampi** [reply]
I’m assuming your EV Charger cannot be controlled via home assistant. If it can be, then I would make an automation that would turn of the EV charger when your battery hits Min SoC and then another automation to turn the charger back on when generating excess power

If that’s not an option, I’ve heard of people installing a contactor switch, connected to a smart plug. Which will allow you do the suggested automations above
